velox/expression/tests/ExpressionRunnerTest.cpp

```diff
@@ -181,9 +188,17 @@ int main(int argc, char** argv) {
     sql = restoreStringFromFile(FLAGS_sql_path.c_str());
     VELOX_CHECK(!sql.empty());
   }
+  if (FLAGS_engine == "spark") {
+    functions::sparksql::registerFunctions("");
+  } else if (FLAGS_engine == "presto") {
+    functions::prestosql::registerAllScalarFunctions();
+    aggregate::prestosql::registerAllAggregateFunctions();
```

Comment:
Do we need to register aggregate functions for expression fuzzer ?

Reply:
In Velox's current state, I believe the answer is no.

Taking a look at ExpressionFuzzerTest.cpp, PrestoSQL fuzzer doesn't seem to include aggregate tests. 

https://github.com/facebookincubator/velox/blob/5e56843165cd3f03ebf0ddd16196507e47ede4bb/velox/expression/tests/ExpressionFuzzerTest.cpp#L32-L59

So if we view this executables use as to only to debug Fuzzer failures, then no I would say it's best to exclude it. Additionally, velox_aggregates isn't even linked to velox_functions_prestosql or anything that velox_expression_fuzzer_test links to, so when you try to run sum for example, you get a core dump.

```
I0122 11:04:04.415263 1856286 ExpressionRunnerTest.cpp:130] File for complex_constant_path not found.
terminate called after throwing an instance of 'facebook::velox::VeloxUserError'
  what():  Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: Scalar function doesn't exist: sum.
Retriable: False
Function: resolveScalarFunctionType
File: /home/sdp/pinterest/velox/velox/parse/TypeResolver.cpp
Line: 96
Stack trace:
Stack trace has been disabled. Use --velox_exception_user_stacktrace_enabled=true to enable it.

*** Aborted at 1705950244 (Unix time, try 'date -d @1705950244') ***
*** Signal 6 (SIGABRT) (0x3e8001c531e) received by PID 1856286 (pthread
```

The only reasons I can think for keeping it are:

1. If aggregates will eventually be included in the fuzzer
2. Or if Velox plans to add a way of creating custom on-disk vector representations. I have a personal local implementation of this as I've found it very helpful in my development, but I don't see a reason why Velox would ever support this by default

Reply:
Thanks Cody,

1. We already have a fuzzer for aggregations, thus i think we can skip on registering aggregates.
2. Velox does have an on disk representation for Vectors, used purely for debugging (See VectorSaver) .